File: Problem 07.py
import statistics
from scipy.stats import skew, kurtosis

def calculate_stats_python(data):
  mean = statistics.mean(data)
  median = statistics.median(data)
  mode = statistics.mode(data)
  sd = statistics.stdev(data)
  skewness = skew(data)
  kurtosis_value = kurtosis(data)

  print("Mean:", mean)
  print("Median:", median)
  print("Mode:", mode)
  print("Standard Deviation:", sd)
  print("Skewness:", skewness)
  print("Kurtosis:", kurtosis_value)


# Calculate statistics in Python
data_python = list(range(1, 51))
calculate_stats_python(data_python)

File: Problem 07.r
calculate_stats <- function(data) {
   mean <- mean(data)
   median <- median(data)
   mode <- mode(data)
   sd <- sd(data)
   skew <- skewness(data)
   kurt <- kurtosis(data)
   
   cat("Mean:", mean, "\n")
   cat("Median:", median, "\n")
   cat("Mode:", mode, "\n")
   cat("Standard Deviation:", sd, "\n")
   cat("Skewness:", skew, "\n")
   cat("Kurtosis:", kurt, "\n")
}

# Generate 1st 50 natural numbers
data <- 1:50

# Calculate statistics in R
calculate_stats(data)
